truffle.js

```diff
@@ -1,34 +1,30 @@
-/*
- * NB: since truffle-hdwallet-provider 0.0.5 you must wrap HDWallet providers in a
- * function when declaring them. Failure to do so will cause commands to hang. ex:
- * ```
- * mainnet: {
- *     provider: function() {
- *       return new HDWalletProvider(mnemonic, 'https://mainnet.infura.io/<infura-key>')
- *     },
- *     network_id: '1',
- *     gas: 4500000,
- *     gasPrice: 10000000000,
- *   },
- */
+const HDWalletProvider = require("@truffle/hdwallet-provider");
 
-const HDWalletProvider = require("truffle-hdwallet-provider-privkey")
-privKey = ''
+require('dotenv').config()
+
+const token = process.env.INFURA_TOKEN
+const mnemonic = process.env.MNEMONIC
 
 module.exports = {
   // See <http://truffleframework.com/docs/advanced/configuration>
   // to customize your Truffle configuration!
   networks: {
     development: {
       host: "127.0.0.1",
-      port: 9545,
+      port: 8545,
```

Comment:
Huh weird, I wonder why I had this...

Anyway, +1 to changing it back to the standard port.